src/drivers/cmakeDriver.ts

```diff
@@ -1837,7 +1837,9 @@ export abstract class CMakeDriver implements vscode.Disposable {
         testPreset: preset.TestPreset | null,
         preferredGenerators: CMakeGenerator[]) {
         this._useCMakePresets = useCMakePresets;
-        log.debug(`Initializating base driver using ${useCMakePresets ? 'preset' : 'kit'}`);
+        const initBaseDriverWithPresetLoc = localize("init.driver.using.preset", "Initializing base driver using preset");
+        const initBaseDriverWithKitLoc = localize("init.driver.using.kit", "Initializing base driver using kit");
+        log.debug(`${useCMakePresets ? initBaseDriverWithPresetLoc : initBaseDriverWithKitLoc}`);
```

Comment:
Not asking for a change, but is the expectation in this extension that all log messages in are localized (even at high logging levels like debug which are probably intended for extension developers)?

Reply:
Chatted offline. Yes this is the expectation since it's open source, it could be useful for those not in english.